schema_based_on_diagram.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE patients (
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(255),
+  date_of_birth DATE,
+  PRIMARY KEY(id)
+  );
+
+CREATE TABLE medical_histories (
+     id INT GENERATED ALWAYS AS IDENTITY,
+     admitted_at timestamp,
+     patient_id INT,
+     status VARCHAR(100),
+     FOREIGN KEY (patient_id) REFERENCES patients(id),
```

Comment:
- [ Required ] Great job adding the foreign key to this table, however, you forgot to add an `index` to the foreign key column. Please consider adding an index on all the foreign key columns as specified in the project requirements as shown below.

![Screenshot from 2023-01-24 14-51-02](https://user-images.githubusercontent.com/16392046/214284683-6a487358-673c-4e8c-b976-9147d16cdf3f.png)

You can achieve this by using the SQL query syntax below.
```SQL
  CREATE INDEX ON medical_histories (patient_id);
```